modules/core/src/test/scala/compiler/CompilerSpec.scala

```diff
@@ -381,7 +386,12 @@ final class CompilerSuite extends CatsSuite {
 
     val res = QueryParser.parseText(query)
 
-    assert(res == Ior.Left(mkOneError("Malformed query")))
+    val error =
+      """Parse error at line 2 column 29
+        |        character(id: "1000" {
+        |                             ^""".stripMargin
```

Comment:
Oh! This test was _supposed_ to be for a missing closing `}` right at the end. I'd missed that that was being preempted by the missing closing `)` on the parameter block.

Reply:
😂 I had wondered why they were the same!